File: python/ex1/main.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        return f"{self.ano} {self.marca} {self.modelo}"

carro1 = Carro("Toyota", "Corolla", 2020)
carro2 = Carro("Honda", "Civic", 2021)
carro3 = Carro("Ford", "Fiesta", 2019)

for carro in [carro1, carro2, carro3]:
    print(carro.exibir_detalhes())

File: golang/ex1/main.go
package main

import (
    "fmt"
)

type Carro struct {
    Marca  string
    Modelo string
    Ano    int
}

func main() {
    carro1 := Carro{"Toyota", "Ethios", 2020}
    carro2 := Carro{"Honda", "City", 2021}
    carro3 := Carro{"Ford", "Ka", 2019}

    fmt.Println(carro1)
    fmt.Println(carro2)
    fmt.Println(carro3)
}
